stl/inc/stacktrace

```diff
@@ -339,13 +339,12 @@ _NODISCARD string to_string(const basic_stacktrace<_Alloc>& _St) {
     return _Result;
 }
 
-_EXPORT_STD template <class _CharT, class _Traits>
-basic_ostream<_CharT, _Traits>& operator<<(basic_ostream<_CharT, _Traits>& _Os, const stacktrace_entry& _Fx) {
+_EXPORT_STD ostream& operator<<(ostream& _Os, const stacktrace_entry& _Fx) {

```

Comment:
Missing `inline` on both of these non-template functions.

Reply:
Tests are failing because `ostream` is incomplete here. We _could_ include `<ostream>` - and have folks who use `<stacktrace>` but not iostreams pay for something they don't use - or artificially templatize this function to preserve the `#include` status quo (a la `template <int = 0>`). I'm leaning toward the latter right now.